tests/test_multiplexer.py

```diff
@@ -163,7 +163,8 @@ def test_multiplexer_disconnect_all_raises_error():
     multiplexer.disconnect()
 
 
-def test_multiplexer_disconnect_one_raises_error_many_connections():
```

Comment:
This is the other test that caused the message warnings like:
```
Task was destroyed but it is pending!
task: <Task pending coro=<Multiplexer._receiving_loop() running at /home/marcofavorito/workfolder/agents-aea/aea/mail/base.py:545> wait_for=<Task pending coro=<DummyConnection.receive() running at /home/marcofavorito/workfolder/agents-aea/tests/conftest.py:119> wait_for=<Future cancelled> cb=[<TaskWakeupMethWrapper object at 0x7fe9ffd591d0>()]> cb=[_chain_future.<locals>._call_set_state() at /usr/lib/python3.7/asyncio/futures.py:355]>
Task was destroyed but it is pending!
task: <Task pending coro=<DummyConnection.receive() running at /home/marcofavorito/workfolder/agents-aea/tests/conftest.py:119> wait_for=<Future cancelled> cb=[<TaskWakeupMethWrapper object at 0x7fe9ffd591d0>()]>
```

this test needs many more changes on `aea/mail/base.py` in order to avoid warnings.
The problem is that, as we decided, the `multiplexer.disconnect()` tries to disconnect the connections and, even if it doesn't manage to do it, stops the multiplexer loop, leaving the connections whose `disconnect()` failed to be in an undefined state.

Reply:
fixed with the `break` statements. See above